elements/lisk-p2p/src/peer/inbound.ts

```diff
@@ -27,14 +26,20 @@ import { SCServerSocket } from 'socketcluster-server';
 
 export const EVENT_CLOSE_INBOUND = 'closeInbound';
 export const EVENT_INBOUND_SOCKET_ERROR = 'inboundSocketError';
+export const EVENT_PING = 'ping';
```

Comment:
I changed it to event `response` instead of a `PONG` event due to @jondubois comment regarding a security issue. An attacker can potentially spam `PONG` events and fake a low latency, whereas with the event response it is explicitly coupled with the `PING` event and cannot be faked as easily.